fideslog/api/config.py

```diff
@@ -151,6 +152,17 @@ class Config:
         env_prefix = f"{ENV_PREFIX}LOGGING_"
 
 
+class SecuritySettings(Settings):
+    """Configuration to secure API endpoints."""
+
+    access_token: str = Field("fides", exclude=True, min_length=1)
```

Comment:
Thanks for adding this. Should `min_length` be something more realistic?

Reply:
The goal was just to guarantee that it isn't `""`. I don't see this being a long term authentication solution, so I probably won't change it.